hs_core/page_processors.py

```diff
@@ -118,6 +118,11 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
     topics = list(topics)  # force QuerySet evaluation
     content_model.update_relation_meta()
     creators = content_model.metadata.creators.all()
+    has_active_creators = False
+    for creator in creators:
+        if creator.is_active:
```

Comment:
This raises an exception if the first creator is not a hydroshare user. You can test by rearranging/re-ordering the authors in UI making the non-hydroshare user as the first author.

Reply:
Correct, thank you. Modified to catch the exception now that I understand better...